ecr/v_func.ecr

```diff
@@ -0,0 +1,20 @@
+    private macro _register_vfunc_<%= vfunc.name %>
+      private def self._vfunc_<%= vfunc.name %>(_this : Pointer(Void), <% generate_lib_args(io, vfunc) %>) : <%= return_type %>
+        <%- write_implementations(io) -%>
+        <%-= '\n' unless vfunc.args.empty? -%>
```

Comment:
Better to just add an extra line break and let `crystal tool format` do the job, so there's less noise in the .ecr file.

Reply:
This is inside a macro, `crystal tool format` will not format anything here.
Here's an example what will be generated:
```crystal
    private macro _register_vfunc_get_intrinsic_width
      private def self._vfunc_get_intrinsic_width(_this : Pointer(Void), ) : Int32
        LibGObject
          .g_object_get_qdata(_this.unsafe_as(Pointer(Void)), GICrystal::INSTANCE_QDATA_KEY)
          .as(self)
          .vfunc_get_intrinsic_width()
      end

      def self._install_iface_Gdk__Paintable(type_struct : Pointer(LibGObject::TypeInterface)) : Nil
        vfunc_ptr = (type_struct.as(Pointer(Void)) + 40).as(Pointer(Pointer(Void)))
        vfunc_ptr.value = (->_vfunc_get_intrinsic_width(Pointer(Void))).pointer
        previous_def
      end
    end

    private macro _register_vfunc_snapshot
      private def self._vfunc_snapshot(_this : Pointer(Void), lib_snapshot :  Pointer(Void), lib_width :  Float64, lib_height :  Float64, ) : Void
# Generator::BuiltInTypeArgPlan
snapshot=Gdk::Snapshot.new(lib_snapshot, :none)
width=lib_width
height=lib_height

        LibGObject
          .g_object_get_qdata(_this.unsafe_as(Pointer(Void)), GICrystal::INSTANCE_QDATA_KEY)
          .as(self)
          .vfunc_snapshot(snapshot, width, height)
      end

      def self._install_iface_Gdk__Paintable(type_struct : Pointer(LibGObject::TypeInterface)) : Nil
        vfunc_ptr = (type_struct.as(Pointer(Void)) + 16).as(Pointer(Pointer(Void)))
        vfunc_ptr.value = (->_vfunc_snapshot(Pointer(Void), Pointer(Void), Float64, Float64)).pointer
        previous_def
      end
    end
```

Notice the comma at the end of `_vfunc_get_intrinsic_width`'s arguments and the implementations having no identation (after `crystal tool format`)